tests/Unit/PointwiseFunctions/AnalyticSolutions/Xcts/Test_Schwarzschild.cpp

```diff
@@ -80,46 +85,75 @@ void test_solution(const double mass,
-      {"conformal_factor_" + py_functions_suffix,
-       "lapse_times_conformal_factor_" + py_functions_suffix,
-       "shift_" + py_functions_suffix,
-       "conformal_factor_gradient_" + py_functions_suffix,
-       "lapse_times_conformal_factor_gradient_" + py_functions_suffix,
-       "shift_strain_" + py_functions_suffix},
-      {{{inner_radius, outer_radius}}}, std::make_tuple(mass), DataVector(5));
-  pypp::check_with_random_values<1>(
-      &SchwarzschildProxy::background_variables, solution, "Schwarzschild",
-      {"conformal_spatial_metric_" + py_functions_suffix,
-       "extrinsic_curvature_trace_" + py_functions_suffix,
-       "extrinsic_curvature_trace_gradient_" + py_functions_suffix,
-       "shift_background"},
-      {{{inner_radius, outer_radius}}}, std::make_tuple(mass), DataVector(5));
-  pypp::check_with_random_values<1>(
-      &SchwarzschildProxy::matter_source_variables, solution, "Schwarzschild",
-      {"energy_density", "stress_trace", "momentum_density"},
-      {{{inner_radius, outer_radius}}}, std::make_tuple(mass), DataVector(5));
-  pypp::check_with_random_values<1>(
-      &SchwarzschildProxy::fixed_source_variables, solution, "Schwarzschild",
-      {"conformal_factor_fixed_source",
-       "lapse_times_conformal_factor_fixed_source", "shift_fixed_source"},
-      {{{inner_radius, outer_radius}}}, std::make_tuple(mass), DataVector(5));
+  {
+    INFO("Random-value tests");
+    const SchwarzschildProxy solution_proxy{mass, coords};
+    const double inner_radius = expected_radius_at_horizon;
+    const double outer_radius = 3. * expected_radius_at_horizon;
+    pypp::check_with_random_values<1>(
+        &SchwarzschildProxy::field_variables, solution_proxy, "Schwarzschild",
+        {"conformal_factor_" + py_functions_suffix,
+         "lapse_times_conformal_factor_" + py_functions_suffix,
+         "shift_" + py_functions_suffix,
+         "conformal_factor_gradient_" + py_functions_suffix,
+         "lapse_times_conformal_factor_gradient_" + py_functions_suffix,
+         "shift_strain_" + py_functions_suffix},
+        {{{inner_radius, outer_radius}}}, std::make_tuple(mass), DataVector(5));
+    pypp::check_with_random_values<1>(
+        &SchwarzschildProxy::background_variables, solution_proxy,
+        "Schwarzschild",
+        {"conformal_spatial_metric_" + py_functions_suffix,
+         "extrinsic_curvature_trace_" + py_functions_suffix,
+         "shift_background"},
+        {{{inner_radius, outer_radius}}}, std::make_tuple(mass), DataVector(5));
+    pypp::check_with_random_values<1>(
+        &SchwarzschildProxy::matter_source_variables, solution_proxy,
+        "Schwarzschild", {"energy_density", "stress_trace", "momentum_density"},
+        {{{inner_radius, outer_radius}}}, std::make_tuple(mass), DataVector(5));
+  }
+  {
+    INFO("Verify the solution solves the XCTS equations");
+    // Once the XCTS equations are implemented we will check here that the
```

Comment:
The 'common variables' from the second commit currently appear to be untested -- I'm fine with the test being rolled into an analytic solution that uses the common variables, but I think the computations performed by these inherited operators should then be tested.
Perhaps this is what you mean by this comment? I would have thought that at least a few of those calculations could be tested reasonably easily by comparing to pointwise GR calculations elsewhere, or with an expanded set of python tests, but perhaps I'm mistaken.

If that's an infeasibly large amount of work, I'll ask that you at least test here a few tags that don't evaluate to exactly `0.0`.

Reply:
I added a bunch of tests, though these quantities actually all evaluate to zero in isotropic Schwarzschild coordinates :D I'll add Painlevé-Gullstrand and isotropic Kerr-Schild coordinates in a follow-up PR where more quantities will be non-trivial. And once #2667 is merged I'll verify the solution actually numerically solves the XCTS equations, as the code comment says.